src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The `href` attribute in the `<link>` tag should point to a CSS file, not an SCSS file. Please ensure the `href` is linked to the compiled CSS file, such as `styles/index.css`.